jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/SlidersTests.java

```diff
@@ -119,12 +115,16 @@ public void minAndMaxSliderTest() {
         minAndMaxSlider.has().value(90);
         minAndMaxSlider.slideHorizontalTo(-50);
         minAndMaxSlider.has().value(-50);
+
+        minAndMaxInput.sendKeys(Keys.CONTROL + "a" + Keys.DELETE + Keys.CONTROL + "a" + Keys.DELETE);
```

Comment:
Type should be change to Input and clear method called instead of this strange code